stored_procedures/definitions/bqml_generate_text.sqlx

```diff
@@ -0,0 +1,203 @@
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+-- Iteratively executes the BQML.GENERATE_TEXT function to ensure all source table rows have generated output in the destination table, 
+-- handling potential retryable errors gracefully along the way. 
+-- Any rows already present in the destination table are ignored, so this procedure is safe to call multiple times.
+
+-- @param STRING source_table : The full path of the BigQuery table containing the input prompts for text generation. Path format - "project.dataset.table" or "dataset.table"
+-- @param STRING target_table : The full path of the BigQuery table where the generated text will be stored. This table will be created if it does not exist.
+-- @param STRING ml_model : The full path of the text model to be used.
+-- @param STRING prompt_column : The name of the column in the `source_table` containing the text promt.
+-- @param ARRAY<STRING> key_columns : An array of column names in the `source_table` that uniquely identify each row. '*' is not a valid value.
+-- @param STRING options_string : A JSON string containing additional optional parameters for the text generation process. Set to '{}' if you want to use defaults for all options parameters.
+-- A sample fully-filled JSON option string would look like:
+-- '{
+--   "batch_size": 500,
+--   "termination_time_secs": 43200,
+--   "source_filter": "LENGTH(text) < 1000",
+--   "projection_columns": ["type", "text"],
+--   "ml_options": "STRUCT(0.5 as temperature)"
+-- }'
+
+-- The parameters within the options string are documented below as:
+-- INT64 batch_size : The number of rows to process in each child job during the procedure. A larger value will reduce the overhead of multiple child jobs, but needs to be small enough to complete in a single job run. Defaults to 1000.
+-- INT64 termination_time_secs : The maximum time (in seconds) the script should run before terminating. Defaults to 82800 (23 hours).
+-- STRING source_filter : An optional filter applied as a SQL WHERE clause to the source table before processing. Defaults to 'TRUE'.
+-- ARRAY<STRING> projection_columns : An array of column to copy from the source table into the destination table. '*' is not a valid value.
+-- STRING ml_options : A JSON string representing additional struct options defined in ML.GENERATE_TEXT. Must be of the form 'STRUCT(...)' 
+CREATE OR REPLACE PROCEDURE ${self()}(source_table STRING, target_table STRING, ml_model STRING, content_column STRING, key_columns ARRAY<STRING>, options_string STRING)
```

Comment:
Please add the comments above as a description attribute within the OPTIONS of the DDL. Please also do this for the other bqml procedure

Reply:
OPTIONS (
description='''Iteratively executes the BQML.GENERATE_TEXT function to ensure all source table rows have generated output in the destination table, 
handling potential retryable errors gracefully along the way. 
Any rows already present in the destination table are ignored, so this procedure is safe to call multiple times.

@param STRING source_table : The full path of the BigQuery table containing the input prompts for text generation. Path format - "project.dataset.table" or "dataset.table"
@param STRING target_table : The full path of the BigQuery table where the generated text will be stored. This table will be created if it does not exist.
@param STRING ml_model : The full path of the text model to be used.
@param STRING prompt_column : The name of the column in the `source_table` containing the text promt.
@param ARRAY<STRING> key_columns : An array of column names in the `source_table` that uniquely identify each row. '*' is not a valid value.
@param STRING options_string : A JSON string containing additional optional parameters for the text generation process. Set to '{}' if you want to use defaults for all options parameters.
A sample fully-filled JSON option string would look like:
'{
  "batch_size": 500,
  "termination_time_secs": 43200,
  "source_filter": "LENGTH(text) < 1000",
  "projection_columns": ["type", "text"],
  "ml_options": "STRUCT(0.5 as temperature)"
}'

The parameters within the options string are documented below as:
INT64 batch_size : The number of rows to process in each child job during the procedure. A larger value will reduce the overhead of multiple child jobs, but needs to be small enough to complete in a single job run. Defaults to 1000.
INT64 termination_time_secs : The maximum time (in seconds) the script should run before terminating. Defaults to 82800 (23 hours).
STRING source_filter : An optional filter applied as a SQL WHERE clause to the source table before processing. Defaults to 'TRUE'.
ARRAY<STRING> projection_columns : An array of column to copy from the source table into the destination table. '*' is not a valid value.
STRING ml_options : A JSON string representing additional struct options defined in ML.GENERATE_TEXT. Must be of the form 'STRUCT(...)' 
''')